source/reference/program/mongostat.txt

```diff
@@ -179,15 +179,21 @@ average operations per second.
    megabytes. This is the total data size at the time of the last
    :program:`mongostat` call.
 
-.. describe:: size
+.. describe:: vsize
 
    The amount of virtual memory in megabytes used by the process at
    the time of the last :program:`mongostat` call.
 
 .. describe:: non-mapped
 
-   The total amount of virtual memory excluding all mapped memory at
+   .. versionchanged:: 3.0.0 
+   Only for :ref:`storage-mmapv1`.
```

Comment:
so weird.  I can't even pull this pull request.  if I do a fake
.. versionchanged:: 3.0.0
This is a test

Mine nags at me, but we seem to have build errors due to michael's debian step and one of the toc's is looking for the core/tag-sharding in  toc-sharded-cluster-data.yaml which doesn't allow for any actual builds.  eh.